src/config.h

```diff
@@ -1,5 +1,7 @@
-/* src/config.h.  Generated from config.h.in by configure.  */
+#include <xapian.h>
+
+/* Xapian::RangeProcesor (and subclasses) was introduced in 1.3.6. Therefore, with Xapian versions before 1.3.6, RXapian uses Xapian::ValueRangeProcessor (and subclasses) for supporting range queries. */
 #if XAPIAN_MAJOR_VERSION >= 2 || XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)
-#define XAPIAN_RANGE_PROCESSOR_SUPPORT
+#define XAPIAN_RANGE_PROCESSOR_SUPPORT && XAPIAN_OP_MAX_SUPPORT
```

Comment:
That won't do what you think, and additionally `OP_MAX` is supported by earlier versions of Xapian, so you should test for it separately.